src/Server/MiddlewareAliasRegistry.php

```diff
@@ -44,11 +44,13 @@ public function registerAliases(array $aliases, bool $replace = false): Middlewa
      */
     public function findNameByAlias(string $alias): string
     {
-        if (!$this->aliasExists($alias)) {
+        [$aliasName, $aliasParams] = $this->parseAlias($alias);
+
+        if (!$this->aliasExists($aliasName)) {
             throw new \InvalidArgumentException("Middleware alias '$alias' has not been registered");
         }
 
-        return $this->registry[$alias];
+        return $this->registry[$aliasName] . (is_null($aliasParams)? '' : ":{$aliasParams}");
```

Comment:
Style: missing space before `?` (and may be === NULL instead of is_null?)

Reply:
Didn't notice, that the project everywhere uses `===`. Will change.